lib/matplotlib/backends/backend_tkagg.py

```diff
@@ -609,6 +609,9 @@ def destroy(*args):
             self.window.update()
         else:
             self.canvas.draw_idle()
+        # Raise the new window.
+        self.canvas.manager.window.attributes('-topmost', 1)
```

Comment:
I wonder if we should make this a method on the ``FigureManager[TkAgg]`` - that way we can re-use in code too.